### PR TITLE
🌱 Increase coverage for web-console-validation server from 51-->100%

### PR DESCRIPTION
**Pull request by dilyar85**
**What does this PR do, and why is it needed?**

This patch increases the code coverage for the `pkg/webconsolevalidation` package from 51 to 100%.

```console
$ ./hack/test.sh ./pkg/webconsolevalidation
...

PASS
coverage: 100.0% of statements
composite coverage: 100.0% of statements
```

**Which issue(s) is/are addressed by this PR?** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

Fixes `NA`

**Are there any special notes for your reviewer**:

**Please add a release note if necessary**:

```release-note
NONE
```

### CODE REVIEW

**Comment by akutz** [comment]
Thank you @dilyar85 !!

FWIW:

> `go test -coverprofile=/tmp/go-code-cover github.com/vmware-tanzu/vm-operator/pkg/webconsolevalidation`

The above can be simplified as:

`hack/test.sh ./pkg/webconsolevalidation`

:)

File: pkg/webconsolevalidation/server.go [hunk 1]
```diff
@@ -20,19 +20,28 @@ import (
 // K8sClient is used to get the webconsolerequest resource from UUID and namespace.
 var K8sClient ctrlclient.Client
 
+// Function variables to allow for mocking in tests.
+var (
```
Comment (akutz) [inline pkg/webconsolevalidation/server.go:24]: Might I ask why you do not just send this into the `InitServer` function as arguments?

**Review by akutz — CHANGES_REQUESTED** [review]
Instead of package-scoped vars for testing that can complicate parallel testing, please send in the vars as arguments to `InitServer` unless there is some compelling reason to not do this.

File: pkg/webconsolevalidation/server.go (hunk 1 above)
Comment (dilyar85) [reply pkg/webconsolevalidation/server.go]: Updated as suggested and also removed the other pkg-scoped var `K8sClient` previously defined in this file.

**Review by akutz — APPROVED** [review]

**Pull request merged**